plugins/contact-resources/src/utils.ts

```diff
@@ -320,7 +320,7 @@ function fillStores (): void {
     const accountPersonQuery = createQuery(true)
 
     const query = createQuery(true)
-    query.query(contact.mixin.Employee, {}, (res) => {
+    query.query(contact.mixin.Employee, { [contact.mixin.Employee + '.active']: { $in: [true, false] } }, (res) => {
```

Comment:
I think you can just use active without `[contact.mixin.Employee + '.active']`, it will be translated into adapter

Reply:
It should yes, let me re-check.

Reply:
Yes it works, will include in next PR.